packages/page-staking/src/Actions/Account/index.tsx

```diff
@@ -45,7 +46,8 @@ function Account ({ className = '', info: { controllerId, destination, destinati
   const stakingAccount = useCall<DeriveStakingAccount>(api.query.staking.ledger, [controllerId]);
   const rewarDestination = useCall<RewardDestination>(api.query.staking.payee, [stashId]);
   destination = rewarDestination && rewarDestination.toString();
-  const role = validators && (validators?.indexOf(stashId) == -1)?'Guarantor':'Validator';
+  const guarantors = useCall<Option<Nominations>>(api.query.staking.guarantors, [stashId])
+  const role = validators && (validators?.indexOf(stashId) == -1)?(guarantors && JSON.parse(JSON.stringify(guarantors)) == null?'':'Guarantor') :'Validator';
```

Comment:
Change to `let role = !isValidator ? (isGuarantor ? "Guarantor" : "None" ) : "Validator"`